### ISSUE TITLE
Could you Update the project from Swift 3.0 to a new version like 4.0?

### ISSUE THREAD

**Issue by davide-areias** [zero2launch/z2l-ios-social-network#3]

**Comment by likhitgarimella** [comment]
Yes, I worked on Swift 5.0 version for this project. Do you need some help in specific?

**Comment by davide-areias** [comment]
> Yes, I worked on the Swift 5.0 version for this project. Do you need some help in specific?

> Hi, I'm very beginner I don't need help in specific, I just wanted to be able to download the full project in Swift5 to mess around with it and learn better, if it's not asking for too much would you link a download for me?
Thanks for responding anyway, I was not expecting an answer ;D.
